README.md

```diff
@@ -29,7 +29,10 @@ Demonstrate calling the method, passing in "young prince" as the argument.
 Write your code here...
 
 ```ruby
-# code here
+def offer_rose person
+  puts "Would you take this rose, #{person.to_s}, in exchange for giving an old beggar woman shelter from the bitter cold?"
+end
+offer_rose 'young prince'
 ```
 
```

Comment:
with string interpolation (`#{}`) you dont need to `to_s`; I believe Ruby calls `to_s` on everything in the interpolation expression